client/state/jetpack-onboarding/schema.js

```diff
@@ -50,5 +50,6 @@ export const jetpackOnboardingSettingsSchema = {
 			],
 		},
 		installWooCommerce: { type: 'boolean' },
+		stats: { type: 'boolean' },
```

Comment:
For consistency with `installWooCommerce`, `activateStats` would make sense here -- but I guess it's already named `stats` in JP now?

Reply:
Yeah, it's called `stats` in Jetpack right now. But if you insist, we can rename it.

Reply:
I'm not insisting 😄